### ISSUE TITLE
getting error while launching desktop Application

### ISSUE THREAD

**Issue by AnkitSinghR** [Lichtblick-Suite/lichtblick#307]
![Screenshot from 2024-12-14 16-57-07](https://github.com/user-attachments/assets/14194ac0-890b-40aa-aec7-f9994c847dd7)
![Screenshot from 2024-12-14 16-56-41](https://github.com/user-attachments/assets/f9ee229c-e275-4fc8-9de8-c131d6f8cbeb)
**Description**

- Version:"1.8.7",
- Platform: ubuntu 22
- Data source (e.g. bag file, mcap file, rosbridge, ROS 1/2 native):

**Steps To Reproduce**

$ yarn desktop:serve  
$ yarn desktop:start
lichtblick is loading but blank screen is showing
attaching screenshot for your reference

**Expected Behavior**

**Comment by laisspportugal** [comment]
Hi @AnkitSinghR

Can you please try to start the application enabling the  LIGBL_ALWAYS_SOFTWARE=1 flag? 
Please let us know the result that you got doing that.
Thanks in advance.

**Comment by go-sakayori** [comment]
@laisspportugal 
Hi, I'm having the same problem with version "1.8.5". 
Just wanted to share that I tried 
```$ LIBGL_ALWAYS_SOFTWARE=1 yarn desktop:start```,
however the same error occurred.
I'm trying to solve it, but do you have any idea where to start from?

**Comment by AnkitSinghR** [comment]
![Uploading Screenshot from 2024-12-16 16-50-35.png…]()
Hi @laisspportugal 
still have the same issue
I have  run the commnad which you have mentioned
$ LIBGL_ALWAYS_SOFTWARE=1 yarn desktop:start
![Screenshot from 2024-12-16 16-47-28](https://github.com/user-attachments/assets/32da939e-b400-4cae-b09c-d8981258dfcb)

**Comment by aneuwald-ctw** [comment]
Hey @AnkitSinghR  thank you for the report, we've identified where the problem is and we're going to work on it.

Best regards.

**Comment by aneuwald-ctw** [comment]
@go-sakayori and @AnkitSinghR the issue should be solved with the merge of #308.

Thanks for spotting it and don't hesitate to reach out again if needed!

**Comment by go-sakayori** [comment]
@aneuwald-ctw Thank you for your quick response. 
I've tried with the latest source, but the same error occurred. 
Here's a screenshot of what happend.
![Screenshot from 2024-12-16 22-05-13](https://github.com/user-attachments/assets/2a7f8d06-5330-4b0f-8ae7-4a1a472eca40)

I tried
- ```rm -rf node_modules```, then ```yarn desktop:serve``` & ```yarn desktop:start```
- ```yarn run clean```, then ```yarn desktop:serve``` & ```yarn desktop:start```
- remove the local git repository and clone -> yarn install

Any other things I should try?
Also should I create a new issue, since this one is closed?
